### ISSUE TITLE
SC1117:  backslash with awk

### ISSUE THREAD

**Issue by henri9813** [koalaman/shellcheck#1364]
#### For bugs
- Rule Id (if any, e.g. SC1000): 
- My shellcheck version (`shellcheck --version` or "online"): 
- [x] I read the issue's wiki page, e.g. https://github.com/koalaman/shellcheck/wiki/SC2086
- [x] I tried on shellcheck.net and verified that this is still a problem on the latest commit

#### For new checks and feature suggestions
- [X shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [X] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh

#!/your/interpreter
command="awk '{print \$3 "\t" \$4}' marks.txt"
```

#### Here's what shellcheck currently says:
```
^-- SC1117: Backslash is literal in "\/". Prefer explicit escaping: "\\/".
```

#### Here's what I wanted or expected to see:

Nothing, in this case i need only one backslash cause i don't want the variable $3 from my script but the $3 from awk

**Comment by koalaman** [comment]
This warning does not happen on the latest commit on shellcheck.net, and is not related to `\$`. Your original code most likely had `\/` in it, which previous versions of shellcheck flagged as being a potential error because `/` does not need escaping in double quotes and leads to the backslash being literal.
